src/routes/nodeinfo.js

```diff
@@ -9,12 +9,11 @@ This Source Code Form is “Incompatible With Secondary Licenses”, as defined
 
 import express from 'express';
 import asyncMiddleware from '../middlewares/async_middleware';
-import getGitCommitHash from '../utils/sys_utils';
 
-export const getNodeInfoHandler = (identityManager) => async (req, res) => {
+export const getNodeInfoHandler = (identityManager) => (req, res) => {
   require('pkginfo')(module, 'version');
   res.status(200).send({
-    commit: await getGitCommitHash(),
+    commit: process.env.GIT_COMMIT,
```

Comment:
Instead of using process.env.GIT_COMMIT directly, you should provide a method in the Config class that will return you this.

Reply:
Good idea :+1: